doc/source/qibo.rst

```diff
@@ -241,23 +501,62 @@ Trotter decomposition. The Hamiltonians represented by this object are sums of
 commuting terms, following the description of Sec. 4.1 of
 `arXiv:1901.05824 <https://arxiv.org/abs/1901.05824>`_.
 
+Trotter hamiltonian
+^^^^^^^^^^^^^^^^^^^
+
 .. autoclass:: qibo.abstractions.hamiltonians.TrotterHamiltonian
     :members:
     :member-order: bysource
 
-
 In addition to these abstract models, Qibo provides the following pre-coded
 Hamiltonians:
 
-.. automodule:: qibo.hamiltonians
-   :members:
-   :member-order: bysource
+.. note::
+    Note that all pre-coded Hamiltonians can be created as either
+    :class:`qibo.abstractions.hamiltonians.Hamiltonian` or
+    :class:`qibo.abstractions.hamiltonians.TrotterHamiltonian` using the ``trotter`` flag.
```

Comment:
I would move this note above the `TrotterHamiltonian` function documentation or at least above the line "... Qibo provides the following pre-coded Hamiltonians", as this refers to the next section.